trucks/functions/increment-completed-trips/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"kargo-back/shared/apigateway"
+	storage "kargo-back/storage/trucks"
+	"net/url"
+
+	"github.com/aws/aws-lambda-go/events"
+	"github.com/aws/aws-lambda-go/lambda"
+)
+
+func apiGatewayHandler(ctx context.Context, request events.APIGatewayProxyRequest) (*events.APIGatewayProxyResponse, error) {
+	body, err := url.ParseQuery(request.Body)
+	if err != nil {
+		return apigateway.LogAndReturnError(err), nil
+	}
+
+	truck, err := storage.LoadTruck(ctx, body.Get("truck_id"))
+	if errors.Is(err, storage.ErrTruckNotFound) {
+		return apigateway.NewErrorResponse(404, err), nil
+	}
+
+	if err != nil {
+		return apigateway.LogAndReturnError(err), nil
+	}
+
+	err = storage.IncrementTruckCompletedTrips(ctx, truck.TruckID)
+	if err != nil {
+		return apigateway.LogAndReturnError(err), nil
+	}
+
+	return apigateway.NewJSONResponse(200, nil), nil
```

Comment:
We should return a message or something here "Successfully incremented" or we could even return the whole truck structure